lib/aggregate/pipeline.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Aggregate
+  class Pipeline
```

Comment:
The more I learn, the more it seems like `Pipeline` should be a Stage.  This class here should be more like a PipelineExecutor (shitty name, but you get the idea.)

The initializer should be passed a klass and an object which you could then use a ValueFactory to render.

Reply:
Agreed, I was thinking the same this AM.

Reply:
I have changed my mind on this one. A pipeline does not behave like a stage. It performs the same to_s transposition however it does not exist as a stage method. Instead a Pipeline is always instantiated explicitly.